### PR TITLE
Clesaec/tdi 42678 release in jenkins

### PR DESCRIPTION
**Pull request by clesaec**
release by jenkins files, [JIRA TDI-42678](https://jira.talendforge.org/browse/TDI-42678)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -57,26 +72,22 @@ spec:
     }
 
     parameters {
-        booleanParam(name: 'PUSH_TO_XTM', defaultValue: false, description: 'Export the project i18n resources to Xtm to be translated. This action can be performed from master or maintenance branches only.')
-        booleanParam(name: 'DEPLOY_FROM_XTM', defaultValue: false, description: 'Download and deploy i18n resources from Xtm to nexus for this branch.')
+        choice(name: 'RunningType',
```
Comment (ypiel-talend) [inline Jenkinsfile]: 'Action' better than 'RunningType' no ?

File: Jenkinsfile [hunk 2]
```diff
@@ -57,26 +72,22 @@ spec:
     }
 
     parameters {
-        booleanParam(name: 'PUSH_TO_XTM', defaultValue: false, description: 'Export the project i18n resources to Xtm to be translated. This action can be performed from master or maintenance branches only.')
-        booleanParam(name: 'DEPLOY_FROM_XTM', defaultValue: false, description: 'Download and deploy i18n resources from Xtm to nexus for this branch.')
+        choice(name: 'RunningType', 
+               choices: [ 'STANDARD', 'PUSH_TO_XTM', 'DEPLOY_FROM_XTM', 'RELEASE' ],
+               description: 'Kind of running : \n PUSH_TO_XTM : Export the project i18n resources to Xtm to be translated. This action can be performed from master or maintenance branches only. \nDEPLOY_FROM_XTM: Download and deploy i18n resources from Xtm to nexus for this branch.')
```
Comment (ypiel-talend) [inline Jenkinsfile]: Explain the two other actions ?

File: Jenkinsfile [hunk 3]
```diff
@@ -222,6 +211,36 @@ spec:
...
+                    expression { BRANCH_NAME.startsWith('maintenance/') }
+                }
+            }
+            steps {
+            	withCredentials([gitCredentials, nexusCredentials]) {
+					container('main') {
+                		
+						sh """
+						    git config --global push.default current
+						    git checkout ${env.BRANCH_NAME}
```
Comment (ypiel-talend) [inline Jenkinsfile]: Not really needed done by jenkins

File: Jenkinsfile [hunk 4]
```diff
@@ -222,6 +211,36 @@ spec:
...
+						    mvn -B -s .jenkins/settings.xml release:clean release:prepare
+						    if [[ \$? -eq 0 ]] ; then
+						        PROJECT_VERSION=\$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout)
+						    	mvn -B -s .jenkins/settings.xml -Darguments='-Dmaven.javadoc.skip=true -DskipTests' release:perform
```
Comment (ypiel-talend) [inline Jenkinsfile]: Don't skip tests for release

File: Jenkinsfile (hunk 1 above)
Comment (clesaec) [reply Jenkinsfile]: Yes, changing

File: Jenkinsfile (hunk 2 above)
Comment (clesaec) [reply Jenkinsfile]: ok

File: Jenkinsfile (hunk 4 above)
Comment (clesaec) [reply Jenkinsfile]: correction done

File: Jenkinsfile (hunk 3 above)
Comment (clesaec) [reply Jenkinsfile]: erased

**Pull request merged**